ApolloExtra.lua

```diff
@@ -2538,10 +2746,10 @@ OlympusLootData.ApolloUpgrade = {
 				PreEventFunctionName = "BoonInteractPresentation", PreEventFunctionArgs = { PickupWait = 1.0, },
 				RequiredTrait = "ForceApolloBoonTrait",
 
-				{ Cue = "/VO/Apollo_0197",
+				{ Cue = "/VO/Apollo_0080",
 					StartSound = "/Leftovers/World Sounds/MapZoomInShort", UseEventEndSound = true,
-					Emote = "PortraitEmoteAffection",
-					Text = "Why, you have my Eternal Rose, don't you? I am so very touched that you'd think first of me among my mighty kin. It pleases me to know I'm in your thoughts!" },
+					Emote = "PortraitEmoteMusical",
+					Text = "I am flattered that you brought my Harmonious Harp with you. I knew we shared a special bond, Zagzag." },
```

Comment:
"I am flattered that you brought my Shining Lyre with you. I knew we shared a special bond, Zagzag."